plexapi/base.py

```diff
@@ -170,7 +170,14 @@ def _manuallyLoadXML(self, xml, cls=None):
         elem = ElementTree.fromstring(xml)
         return self._buildItemOrNone(elem, cls)
 
-    def fetchItems(self, ekey, cls=None, container_start=None, container_size=None, maxresults=None, **kwargs):
+    def fetchItems(self,
+                   ekey,
+                   cls=None,
+                   container_start=None,
+                   container_size=None,
+                   maxresults=None,
+                   params=None,
+                   **kwargs):
```

Comment:
black formatting

```suggestion
    def fetchItems(
        self,
        ekey,
        cls=None,
        container_start=None,
        container_size=None,
        maxresults=None,
        params=None,
        **kwargs,
    ):
```
_Originally posted  in https://github.com/pkkid/python-plexapi/pull/1302#discussion_r1452798533_